src/runtime/graph.cc

```diff
@@ -1399,26 +1648,115 @@ T SearchHelper::graph_cost(Graph const *graph,
 
   check_matches_graph<T>(graph, result, sink.node);
 
+  // This is where we really add the costs of an operator
   if (include_sink_compute_time) {
+    // Sink node costs
     CostMetrics metrics =
         this->model->simulator->measure_operator_cost(sink.node.ptr, sink.view);
-    this->logger->debug() << "Sink node cost: "
+
+    // Adjust operator memory usage
```

Comment:
Most of the added code is to output very detailed spew() messages. They look like a lot because our code formatting uses length 80 (which is a bit too narrow for modern monitors, but it's too late to change).

Reply:
The line breaks for this code don't seem to be adding all that many lines, just feels like there's enough going on here that it should be broken up. Doesn't need to be done in this PR